js/src/product-feed/issues-table-card/issues-solved.js

```diff
@@ -0,0 +1,54 @@
+/**
+ * External dependencies
+ */
+import { capitalize } from 'lodash/string';
```

Comment:
💅 
Using `from 'lodash/string'` will increase the bundle size by 3.4 KB after gzipped.

Just noticed that we should import lodash functions by `from 'lodash'` without any subpath. Therefore, it would use the lodash injected by `@wordpress/dependency-extraction-webpack-plugin` in runtime.

I will open another PR to change all uses of `import isEqual from 'lodash/isEqual` in other places. Then the bundle size could be reduced by about 4 KB after gzipped.

Reply:
> I will open another PR to change all uses of `import isEqual from 'lodash/isEqual` in other places. Then the bundle size could be reduced by about 4 KB after gzipped.

PR opened: #1197

Reply:
Updated here:

https://github.com/woocommerce/google-listings-and-ads/pull/1196/commits/f72fa4f23758c31147499fe1a2607a7b805224c5

Since now we are hardcoding the translations, `capitalize` is not necessary anymore